### DISCUSSION TITLE
Busca de Clientes na Data-Table

### DISCUSSION THREAD

**Discussion by FlaviusR** [RamonSilva20/mapos#2048] [Q&A]
Bem, no sistema quando havia pelo menos uns 100 clientes, a busca 'ajax' funcionava normal.
Agora, o sistema possui mais 720 clientes, e busca nao retorna em tempo real na primeira página.
Tem que correr a paginação, por diversas vezes até tentar localizar tal cliente.
![image](https://user-images.githubusercontent.com/7925068/215779406-0c15dedd-edd5-4909-b1c3-8789ae3d7533.png)

NO exemplo acima, estou buscando por FLAVIO, há **6 Flavios cadastrados**, mas ele me retornou um só, estando na paginação 3.
Ja no exemplo abaixo, para buscar outro FLAVIO, tive que ir na paginação 2.
![image](https://user-images.githubusercontent.com/7925068/215780023-d4878ead-a60d-49ed-8924-994977e3ac03.png)

Alguém ja passou por este problema quando há muitos clientes?
Se puderem dar um norte, agradeço.
Já trabalhei com PHP, mas, o CI ainda é um desafio pra mim.

**Comment by felipe8764** [comment]
Estou na mesma situação. conseguiu resolver?

**Comment by juliolobo** [comment]
A busca por ajax realmente não funciona bem quando temos vários clientes, por isso implementei um outra busca que busca diretamente no banco de dados. Atualizem seus mapos
